doc/writing.md

```diff
@@ -112,8 +112,7 @@ applications like Slack or GitHub):
 :tada:
 ```
 
-Most of the existing emoji are listed on this cheat sheet:
-http://www.emoji-cheat-sheet.com/.
+Refer [this](http://www.emoji-cheat-sheet.com/) emoji cheat sheet.
```

Comment:
I think "refer" needs to have a "to" after it, have a look at these examples: http://www.dictionary.com/browse/refer